src/components/Transcriber.tsx

```diff
@@ -368,6 +368,13 @@ export function Transcriber(props: IProps) {
       direction: projData?.fontDir as any,
       cursor: trBusy || loading ? 'progress' : 'default',
     });
+    if (transcriptionRef.current) {
```

Comment:
Maybe this would be better in showTranscription?  moving the cursor every time the busy changes seems dangerous

Reply:
When I put it there, it has no effect because this code runs after and by setting the style (i.e. changing to the busy cursor) it resets the position to the beginning. I changed the code so that it only positions to the end if it is at the beginning.